### ISSUE TITLE
did-key does not support services

### ISSUE THREAD

**Issue by nklomp** [decentralized-identity/.well-known#82]
The examples in the spec use did:key everywhere, but did:key only supports verificationMethod/keys and not services.

So the examples in the spec cannot satisfy the actual requirements from the spec as you cannot get the service with the origin in the DID.

Probably makes sense to use another DID method for the examples as to not confuse people.

**Comment by OR13** [comment]
I assume this is referring to the well known did configuration? AFAIK, services are not a requirement for that.

**Comment by OR13** [comment]
could add example: https://did.actor and then use it here... PRs welcome.

**Comment by jceb** [comment]
We'll go with did:web:example.org since it's in line with other RFCs.

**Comment by jceb** [comment]
@nklomp the did:key example in the spec is compatible with the spec, because the linking from the DID document is optional (`MAY`), see https://identity.foundation/well-known-did-configuration/resources/did-configuration/#linked-domain-service-endpoint

**Comment by nklomp** [comment]
Yes, but that still makes it a terrible example of course. Not really the answer I was expecting tbh (after 3 years)

**Comment by jceb** [comment]
That's right. Take a look at PR #110 that will fix the issue.
